docker/.env

```diff
@@ -60,10 +60,14 @@ MF_JAEGER_FRONTEND=16686
 MF_JAEGER_COLLECTOR=14268
 MF_JAEGER_CONFIGS=5778
 MF_JAEGER_URL=http://jaeger:14268/api/traces
+MF_JAEGER_TRACE_RATIO=1.0
 
 ## Call home
 MF_SEND_TELEMETRY=true
 
+## Postgres
+MF_POSTGRES_MAX_CONNECTIONS=100000
```

Comment:
Why we should have 100,000 connections by default ?

Reply:
Reduced to 100 i.e default